test/model/src/core/verify.cpp

```diff
@@ -132,8 +149,10 @@ kv_table_verifier::verify(WT_CONNECTION *connection, kv_checkpoint_ptr ckpt)
                 std::cout << "Verification: key = " << key << ", value = " << value << std::endl;
             if (!model_cursor.verify_next(key, value)) {
                 std::ostringstream ss;
+                std::pair<data_value, data_value> prev = model_cursor.get_prev();
```

Comment:
`auto prev = ...`?

Reply:
Sure. Different people have different preferences :)

Reply:
Thanks. Feel free to push back if I request stuff like this and it's not your preference :-)